src/v/cluster/partition_balancer_backend.cc

```diff
@@ -43,12 +45,13 @@ static constexpr std::chrono::seconds add_move_cmd_timeout = 10s;
 partition_balancer_backend::partition_balancer_backend(
   consensus_ptr raft0,
   ss::sharded<controller_stm>& controller_stm,
+  ss::sharded<features::feature_table>& feature_table,
   ss::sharded<partition_balancer_state>& state,
   ss::sharded<health_monitor_backend>& health_monitor,
   ss::sharded<partition_allocator>& partition_allocator,
   ss::sharded<topics_frontend>& topics_frontend,
   ss::sharded<members_frontend>& members_frontend,
-  config::binding<model::partition_autobalancing_mode>&& mode,
+  config::sanctioning_binding<model::partition_autobalancing_mode>&& mode,
```

Comment:
nit: a commit message is highlighting `kafka/server` while this commit has nothing to do with the server implementation, the next one is the same